File: ninth grade/Pygame/Pascal/cekGenap.cpp
#include <cstdio>

bool cekGenap(int number){
  if (number % 2 == 0){
    return true;
  } else {
    return false;
  }
}

int main(){
  int myNumber = 10;
  if (cekGenap(myNumber))
  {
    printf('Genap\n')
  } else {
    printf('ganjil\n')
  }
}

File: ninth grade/Pygame/Pascal/cekGenap.py

def cekGenap(number):
    if number % 2 == 0:
        return True
    else:
        return False

def main():
    myNumber = 10
    if cekGenap(myNumber):
        print('Genap!')
    else:
        print('Ganjil!')

main()    
